build/containers/Dockerfile.agent

```diff
@@ -9,7 +9,16 @@ RUN echo "Using Rust binaries from ${CROSS_BUILD_TARGET}/${BUILD_TYPE}"
 # Link the container to the Akri repository
 LABEL org.opencontainers.image.source https://github.com/project-akri/akri
 
-RUN apt-get update && apt-get install -y --no-install-recommends libssl-dev openssl && apt-get clean
+RUN apt-get update && apt-get install -y --no-install-recommends libssl-dev openssl curl ca-certificates && apt-get clean
+
 COPY ./target/${CROSS_BUILD_TARGET}/${BUILD_TYPE}/agent /agent
+
+# Install crictl
+RUN VERSION="v1.17.0" && \
+    curl -L https://github.com/kubernetes-sigs/cri-tools/releases/download/$VERSION/crictl-$VERSION-linux-amd64.tar.gz --output crictl-$VERSION-linux-amd64.tar.gz && \
+    tar zxvf crictl-$VERSION-linux-amd64.tar.gz -C /usr/local/bin && \
+    rm -f crictl-$VERSION-linux-amd64.tar.gz && \
+    apt-get remove -y curl ca-certificates && apt-get clean
```

Comment:
Can we merge the two RUNs? Otherwise curl/ca-certs will be part of the akri layers and contribute to the overall size.

Reply:
I condensed them in 75f8467b8652aff8c72dc222ed6fd3c161832eea but it didn't decrease the size at all interestingly.  Still 161 MB

Reply:
Interesting. Do we know ca-certs and curl were not in the initial image to begin with or installed as dependencies of the other packages we are installing?

Reply:
I am pretty sure they werent there because it would not install until i added them